### ISSUE TITLE
When a user submits the review form on the frontend, display a loading spinner and dont let them click the button again

### ISSUE THREAD

**Issue by sheenarbw** [Umuzi-org/Tilde#26]
There isn't enough feedback on the frontend. Basically if a user tried to submit a review then it can seem like nothing happened.  Some users then do rapid-fire clicking on the form submission button. This sends multiple api calls to the backend and makes a tonne of duplicate reviews on a project card. 

What we need is:
1. A way to detect that the api call is still underway
2. the submit button to be deactivated while the api call is underway
3. A loading spinner 

To get this right it would be useful to know how to monitor api calls.

Basically inside the state you can see a funch of things THAT_LOOK_LIKE_THIS. Eacvh of those represents an api call log.
You'll want to monitor the last element of `state.CARD_ADD_REVIEW`.
